nshlib/nsh_proccmds.c

```diff
@@ -613,29 +614,54 @@ int cmd_ps(FAR struct nsh_vtbl_s *vtbl, int argc, char **argv)
 #ifndef CONFIG_NSH_DISABLE_KILL
 int cmd_kill(FAR struct nsh_vtbl_s *vtbl, int argc, char **argv)
 {
+/* kill will send SIGTERM to the task in case no signal is selected by
```

Comment:
You need to space in the line bellow "{" to align correctly

Reply:
done. file passed checkpatch.sh